src/main/resources/changelog/changes/v1.0.0/create-holdings-table.xml

```diff
@@ -0,0 +1,42 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+  <changeSet id="task-001@@create-holdings-table" author="sherzod">
+    <createTable tableName="holdings">
+      <column name="id" type="uuid"/>
+      <column name="updated_at" type="timestamptz"/>
+      <column name="credentials_id" type="uuid">
+        <constraints nullable="false"/>
+      </column>
+      <column name="vendor_id" type="varchar(10)"/>
+      <column name="package_id" type="varchar(10)"/>
+      <column name="title_id" type="varchar(10)"/>
```

Comment:
Are you sure that they should be varchar? As I remember these IDs are integers.

Reply:
done